governator-core/src/main/java/com/netflix/governator/event/ApplicationEvent.java

```diff
@@ -0,0 +1,10 @@
+package com.netflix.governator.event;
+
+/**
+ * Marker interface for application events to be published
+ * to subscribed consumers.
+ */
+public interface ApplicationEvent {
```

Comment:
I'm not a big fan of marker interfaces.  I think it's preferable to have this as an annotation on the handler method.